pkgs/tools/graphics/ploticus/default.nix

```diff
@@ -10,25 +10,32 @@
 , runCommand
 }:
 
-stdenv.mkDerivation (finalAttrs: rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "ploticus";
   version = "2.42";
 
   src = fetchurl {
-    url = "mirror://sourceforge/ploticus/ploticus/${version}/ploticus${lib.replaceStrings [ "." ] [ "" ] version}_src.tar.gz";
+    url = "mirror://sourceforge/ploticus/ploticus/${finalAttrs.version}/ploticus${lib.replaceStrings [ "." ] [ "" ] finalAttrs.version}_src.tar.gz";
     sha256 = "PynkufQFIDqT7+yQDlgW2eG0OBghiB4kHAjKt91m4LA=";
   };
 
   patches = [
     # Replace hardcoded FHS path with $out.
     ./ploticus-install.patch
 
+    # Set the location of the PREFABS directory.
+    ./set-prefabs-dir.patch
+
     # Use gd from Nixpkgs instead of the vendored one.
     # This is required for non-ASCII fonts to work:
     # https://ploticus.sourceforge.net/doc/fonts.html
     ./use-gd-package.patch
   ];
 
+  postPatch = ''
```

Comment:
Ideally, this would go next to the other hooks, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.

Reply:
Putting this next to the patches seemed natural  to me, but 🤷🏻